prebuilt_swift_static_framework/prebuilt_swift_static_framework.bzl

```diff
@@ -86,24 +125,53 @@ _prebuilt_swift_static_framework = rule(
             cfg = "host",
             executable = True,
         ),
-    ),
+    )),
+    fragments = ["apple"],
     outputs = {
         "fat_file": "%{name}.fat",
         "output_file": "%{name}.zip",
     },
 )
 
-def prebuilt_swift_static_framework(name, srcs = [], deps = []):
+def prebuilt_swift_static_framework(name, srcs = [], deps = [], **kwargs):
+    """Builds and bundles an iOS Swift static framework for third-party distribution.
+
+    This rule supports building the following:
+        $ bazel build SwiftLibrary (builds a `swift_library` that other targets can depend on)
+        $ bazel build SwiftLibraryFramework (builds an iOS Swift static framework)
+
+    To build for multiple architectures, specify the list of architectures to
+    build the framework with: `--ios_multi_cpus=i386,x86_64,armv7,arm64`. If no
+    architectures are specified, the default is `x86_64`.
+
+    Args:
+      name: Name of the `swift_library` that the framework depends on.
+      srcs: The Swift source files to compile. If no srcs are provided, it's
+          assumed that source files are located in the `Sources` directory.
+      deps: Dependencies of the `swift_library` target being compiled.
+      **kwargs: Additional arguments to pass to this rule, such as `testonly`,
+          `module_name`, `minimum_os_version`, `visibility`, etc.
```

Comment:
See my other comment, but this should remove `etc`, since the comment lists only the valid args.

Reply:
Updated documentation.